benchmark_rag_store.py

```
@@ -0,0 +1,267 @@
+    except ValueError:
+        parser.error("Invalid --sizes format. Please provide comma-separated integers.")
+    if not sizes:
+        parser.error("--sizes must contain at least one positive integer.")
+    if any(size <= 0 for size in sizes):
+        parser.error("--sizes values must all be positive integers.")
+    if args.queries <= 0:
+        parser.error("--queries must be a positive integer.")
+    if args.dim <= 0:
+        parser.error("--dim must be a positive integer.")
+        
+    print(f"============================================================")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)